docs/languages/cpp.md

```diff
@@ -166,7 +166,7 @@ After you have set up the basics of your debugging environment as specified in [
 VS Code supports the following debuggers for C/C++ depending on the operating system you are using:
 
 * **Linux**: GDB
-* **OS X**: LLDB or GDB
+* **Mac**: LLDB or GDB

```

Comment:
Hi, thank you for the PR. The CPP topic is owned by the team who owns the C++ extension and not VS Code. I think they have only tested this on OS X so I don't think they can update this yet to Mac or macOS.